### ISSUE TITLE
Need a mechanism to restrict reimporting old recipes when use import_recipes command.

### ISSUE THREAD

**Issue by suchoudh** [cproctor/cookbook#5]
~/Dropbox/1700_food/cookbook/cookbook$ ./manage.py import_recipes

if I run the command twice the old recipes are reimported making a double entry. 
Either need to stop importing old recipes or need to find if there is a mechanism to revert to latest updated recipe list.

**Comment by cproctor** [comment]
Good point. I usually just drop all the old recipes, effectively storing state in my recipe sources, but that's a bit careless. You could PR, or I'll make an update when I have a minute. Thanks!
